orangecontrib/bio/obiGenialis.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
```

Comment:
I do not think we need to support Python/Orange 2, @JakaKokosar should ask Blaz.